packit/constants.py

```diff
@@ -206,14 +206,17 @@
     "git clone ssh://$YOUR_USER@pkgs.fedoraproject.org/forks/{user}/rpms/{package}.git\n"
     "cd {package}\n"
     "git checkout {branch}\n"
+    "git push origin {branch}\n"
     "```\n"
     "\n---\n\n"
-    "Alternatively, you can locally fetch the source branch of this PR "
-    "from Packit's fork and push it (with a fix) to your fork:\n"
+    "Alternatively, if you already have the package repository cloned, "
+    "you can just fetch the Packit's fork:\n"
     "```\n"
-    "git fetch https://src.fedoraproject.org/forks/{user}/rpms/{package}.git "
-    "refs/heads/*:refs/remotes/{user}/*\n"
-    "git checkout {user}/{branch}\n"
+    "cd {package}\n"
+    "git remote add packit ssh://$YOUR_USER@pkgs.fedoraproject.org/forks/{user}/rpms/{package}.git\n"
+    "git fetch packit refs/heads/{branch}\n"
+    "git checkout {branch}\n"
+    "git push packit {branch}\n"
```

Comment:
:+1: for the push commands. It's simple but one can easily get confused by the fetches and remotes...